tailwind.config.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
   theme: {
     extend: {
       colors: {
+        'teal': '#4FADBE',
```

Comment:
Would there be a name for this color that is closer aligned to the use case on the website?  Like `background-color` or `primary-color`?

I'd like your opinion about nomenclature as I have heard arguments from both side regarding whether to align colors with the business use case on the site or what the color is, in case we want to use it again.

Reply:
Good question. My opinion on this matter is to avoid trying to generalize the name of a class by its use case and would keep it to be `bg-teal`. Here is why and where I came up with this:

- At such early stages of a product, I think trying to formalize a styling pattern is pretty challenging especially if you are doing it with CSS naming. I've found that we end up running into the issue of I want the background color but I want it different so I'll create another style. The spirit of tailwind is to keep them as a utility so you can construct components with them rather than letting CSS handle that for you. 
- I'am in the boat that CSS cannot really live on its own without the underlying HTML. We've seen people do some pretty amazing things with just CSS to change a whole site styling. But as uncle ben from spiderman once said, with great power comes great responsibility. I've found this to inspire more fear into developers from changing styles if they aren't sure where and how it is being used. For that reason, setting the color teal is pretty clear what you are changing.

This is speaking from a person whose seen myself and teams get all tied up and tangled with CSS. This tailwind approach at least today has been very great for inspiring confidence in developers. Long winded way, in saying yeh let's keep it as `teal` :)

Notably - I used the `bg-teal` in the container partial which now if you want to change the container or anything related to it you'd go there :).